tests/api2/jail_plugins.py

```diff
@@ -0,0 +1,291 @@
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    transmission_plugin = results.json()
+
+
+@to_skip
+def test_12_get_transmission_jail_info():
+    global transmission_jail
+    results = GET("/jail/id/transmission")
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    transmission_jail = results.json()
+
+
+@to_skip
+@pytest.mark.parametrize('object', plugins_objects)
+def test_13_verify_transmission_plugin_value_with_jail_value_of_(object):
+    assert transmission_jail[object] == transmission_plugin[object], results.text
+
+
+@to_skip
+def test_14_get_list_of_available_plugins_with_cache():
+    global JOB_ID
+    payload = {
+        "plugin_repository": plugin_repos,
+        "cache": True
+    }
+    results = POST('/plugin/available/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    JOB_ID = results.json()
+
+
+@to_skip
+def test_15_verify_list_of_available_plugins_job_id_is_successfull():
+    global job_results
+    while True:
+        job_results = GET(f'/core/get_jobs/?id={JOB_ID}')
+        job_state = job_results.json()[0]['state']
+        if job_state in ('RUNNING', 'WAITING'):
+            sleep(3)
+        else:
+            assert job_state == 'SUCCESS', job_results.text
+            break
+
+
+@to_skip
+@pytest.mark.parametrize('plugin', plugins_list)
+def test_16_verify_available_plugin_with_want_cache_(plugin):
+    for plugin_info in job_results.json()[0]['result']:
+        if plugin in plugin_info:
+            assert isinstance(plugin_info, list), job_results.text
+            assert plugin in plugin_info, job_results.text
+
+
+@to_skip
+def test_17_stop_transmission_jail():
+    global results
+    payload = {
+        "jail": "transmission",
+        "force": True
+    }
+    results = POST('/jail/stop/', payload)
+    assert results.status_code == 200, results.text
+
+
+@to_skip
+def test_18_wait_for_transmission_plugin_to_be_down():
+    results = GET('/plugin/id/transmission/')
+    while results.json()['state'] == 'up':
+        sleep(1)
+        results = GET('/plugin/id/transmission/')
+        assert results.status_code == 200, results.text
+    assert results.json()['state'] == 'down', results.text
+
+
+@to_skip
+def test_19_start_transmission_jail():
+    global results
+    payload = "transmission"
+    results = POST('/jail/start/', payload)
+    assert results.status_code == 200, results.text
+
+
+@to_skip
+def test_20_wait_for_transmission_plugin_to_be_up():
+    results = GET('/plugin/id/transmission/')
+    while results.json()['state'] == 'down':
+        sleep(1)
+        results = GET('/plugin/id/transmission/')
+        assert results.status_code == 200, results.text
+    assert results.json()['state'] == 'up', results.text
+
+
+@to_skip
+def test_21_stop_transmission_jail_before_deleteing():
```

Comment:
I think we can skip this as stopping has been tested before, we should delete with force  flag enabled which will stop + destroy, so we will test multiple scenarios this way. This would be passed over to jail.

Reply:
We did accomplish that in jail testing.